hls4ml/writer/vivado_writer.py

```diff
@@ -101,6 +101,15 @@ def write_project_cpp(self, model):
         ###################
 
         filedir = os.path.dirname(os.path.abspath(__file__))
+
+        # create project.tcl
+        f = open('{}/project.tcl'.format(model.config.get_output_dir()), 'w')
+        f.write('variable myproject\n')
+        f.write('set myproject "{}"\n'.format(model.config.get_project_name()))
+        f.write('variable backend\n')
+        f.write('set backend "vivado"\n')
+        f.close()
+
```

Comment:
This shouldn't be in this function. Either it goes to its own function, or at least to `write_build_script`